core/templates/services/audio-player.service.ts

```diff
@@ -89,9 +89,22 @@ angular.module('oppia').factory('AudioPlayerService', [
     var _rewind = function(seconds) {
       if (_currentTrack) {
         var currentSeconds = _currentTrack.progress * _currentTrack.duration;
-        var rewindedProgress =
-          (currentSeconds - seconds) / _currentTrack.duration;
-        _currentTrack.progress = rewindedProgress;
+        if (currentSeconds - seconds > 0) {
```

Comment:
Just to confirm: is the intended behaviour in the underflow case to do nothing, rather than jump back to the beginning? Is that what the user expects?

Ditto for _forward(). I can kind of understand not doing it for _forward(), but I'm a bit uncertain about making this choice for _rewind(). What do other video/audio players typically do?

Reply:
Checked the UX for youtube, it starts the audio from the beginning. Made ours's the same way.